app.py

```diff
@@ -1,35 +1,54 @@
+import numpy as np
 import plotly.express as px
 import streamlit as st
 
-from indices.utils import get_heading_names, load_text, get_pair_names, load_percentile_data, load_journal_data
+from indices import utils
```

Comment:
Yep, it has the same effect as `import indices.utils as utils` would! StackOverflow says the mechanism for accomplishing it may be different but no one seems to know why https://stackoverflow.com/a/50943495